sdk/program/src/clock.rs

```diff
@@ -57,7 +57,7 @@ pub const MAX_HASH_AGE_IN_SECONDS: usize = 120;
 
 #[cfg(test)]
 static_assertions::const_assert_eq!(MAX_RECENT_BLOCKHASHES, 300);
-// Number of maximum recent blockhashes (one blockhash per slot)
+// Number of maximum recent blockhashes (one blockhash per slot with a completed block)
```

Comment:
nit: i like `one blockhash per not-skipped slot`. `skipped slot` is more commonly used. i know negation here is a bit awkward...